Open button in a new tab ability for HFG Buttons

mskapusuz opened the pull request:
### Summary
<!-- Please describe the changes you made. -->
* Opening the link in a new tab ability was added to the Button components of the HFG.

**Note to reviewer:**
In our .pot file, I couldn't find a proper translation for the toggle label
https://github.com/Codeinwp/neve/blob/feat/hfg-button-open-blank/header-footer-grid/Core/Components/Button.php#L169

### Will affect visual aspect of the product
<!-- It includes visual changes? -->
NO

### Screenshots <!-- if applicable -->
![Screen Shot 2023-02-26 at 23 15 36](https://user-images.githubusercontent.com/25361626/221434874-4d30b018-ef45-49cc-a2c8-0a9030c63d39.png)


### Test instructions
<!-- Describe how this pull request can be tested. -->
- Install Neve, Neve Pro
- All button components of the HFG should have an "opening the link in a new tab" capability. Note: the default behavior is `false.`

<!-- Issues that this pull request closes. -->
Closes https://github.com/Codeinwp/neve/issues/3848.
<!-- Should look like this: `Closes #1, #2, #3.` . -->

irinelenache commented:
@mskapusuz Tested and here is what i found:

- [ ] On mobile, the behaviour will be inherited from the desktop version. It doesn't matter if it's activated or deactivated, the button will act as the desktop one https://vertis.d.pr/v/WyxhPs
- [ ] In customizer, if i switch to mobile and activate/deactivate the new tab toggle, i'm getting some warnings in console https://vertis.d.pr/i/PQBLSn

mskapusuz commented:
> @mskapusuz Tested and here is what i found:
> 
> * [x]  On mobile, the behaviour will be inherited from the desktop version. It doesn't matter if it's activated or deactivated, the button will act as the desktop one https://vertis.d.pr/v/WyxhPs
> * [x]  In customizer, if i switch to mobile and activate/deactivate the new tab toggle, i'm getting some warnings in console https://vertis.d.pr/i/PQBLSn

@irinelenache thanks for your findings. As I see, I've missed something. I've fixed the logical issues. Now, it should be work as independent by the device, as non-responsive. That's ready for another test round 🚀

mskapusuz commented:
so, the expected behavior is all devices should be sync. let’s say you chosen open new tab for desktop; the mobile and tablet also should be open as new tab. So no matter the device

irinelenache commented:
@mskapusuz Checked again and everything works fine now, thank you 🚀

cristian-ungureanu commented:
@mskapusuz The CSS size limit is exceeded. Can you please have a look here?

mskapusuz commented:
@cristian-ungureanu I've merged development into this branch, and the size issue was fixed this way. To handle that, I've moved the issue to the "todo" column and moved to the "ready to merge" column again because I haven't made any changes or conflict-solving after the QA approval except for the merge.

mskapusuz commented:
@cristian-ungureanu the "add labels to pull request" check is failing since I created this PR before the check was implemented. I think we can ignore it for this PR, but if you want, I can handle it. Thanks!
